modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -527,18 +584,29 @@ private Map<String, Serializable> _toObjectValues(
 	@Reference
 	private GroupLocalService _groupLocalService;
 
+	private boolean _hasPreventTypeRelationship;
+
 	@Reference
 	private ObjectEntryDTOConverter _objectEntryDTOConverter;
 
+	@Reference
+	private ObjectEntryLocalService _objectEntryLocalService;
+
 	@Reference
 	private ObjectEntryService _objectEntryService;
 
 	@Reference
 	private ObjectFieldLocalService _objectFieldLocalService;
 
+	@Reference
+	private ObjectRelationshipLocalService _objectRelationshipLocalService;
+
 	@Reference
 	private ObjectScopeProviderRegistry _objectScopeProviderRegistry;
 
+	private final Set<ObjectRelationship> _preventObjectRelationships =
+		new LinkedHashSet<>();
```

Comment:
Do you need the initialization to be `LinkedHashSet` ? Could it be `new HashSet<>();` ?
I found this article explaining the differences:  https://www.javatpoint.com/hashset-vs-linkedhashset-in-java

Reply:
Oh, follow this article "HashSet is slightly faster than the LinkedHashSet". I will change this